src/models/generated/EnergyShareRequirements.jl

```diff
@@ -0,0 +1,71 @@
+#=
+This file is auto-generated. Do not edit.
+=#
+
+#! format: off
+
+"""
```

Comment:
What is this requirement about? I can't interpret the rationale for this

Reply:
We should add a docstring that explains that this does:
```
sum(P)_eligible >= total_fraction * sum(P)_all
```